### ISSUE TITLE
[Convex mean] try graphical_lasso

### ISSUE THREAD

**Issue by gcattan** [pyRiemann/pyRiemann-qiskit#151]
Learning from POC: https://github.com/pyRiemann/pyRiemann-qiskit/pull/132

We can try to replace Shrinkage by GraphicalLasso for regularization of the empirical covariance (inside mean.py)
https://scikit-learn.org/stable/modules/generated/sklearn.covariance.graphical_lasso.html#sklearn.covariance.graphical_lasso

**Comment by gcattan** [comment]
Note, the main issue is that when using convex mean in association with a log-defined metric for distance, the logarithm of the matrices contains sometimes NaN values.

**Comment by gcattan** [comment]
Investigated within https://github.com/pyRiemann/pyRiemann-qiskit/pull/130.
Graphical_lasso does not provide good results.
